components/card/w-card.vue

```diff
@@ -13,15 +13,13 @@ const props = defineProps({
   flat: Boolean,
 });
 
-const containerClasses = computed(() => ({
-  [ccCard.card]: true,
-  [ccCard.cardShadow]: !props.flat,
-  [ccCard.cardSelected]: !props.flat && props.selected,
-  [ccCard.cardFlat]: props.flat,
-  [props.selected ? ccCard.cardFlatSelected : ccCard.cardFlatUnselected]: props.flat,
-}));
+const containerClasses = computed(() => [
+  ccCard.base,
+  props.flat ? ccCard.flat : ccCard.shadow,
+  props.selected ? (props.flat ? ccCard.flatSelected : ccCard.selected) : props.flat && ccCard.flatUnselected,
```

Comment:
Objects and arrays are combinable (in Vue at least), so sometimes it might be more readable to do this.
```suggestion
  {
    [ccCard.cardSelected]: !props.flat && props.selected,
    [props.selected ? ccCard.cardFlatSelected : ccCard.cardFlatUnselected]: props.flat,
  },
```
But the rules here is really complex, so it might not be any better in this case...? 😅

Reply:
I could also just change it to this: 
```suggestion
  props.selected && !props.flat && ccCard.selected,
  props.selected && props.flat ? ccCard.flatSelected : ccCard.flatUnselected,
```
Because I agree that the combined ternary expression can be confusing to read 😅